### DISCUSSION TITLE
Level 9

### DISCUSSION THREAD

**Discussion by PDNinja333** [nroutasuo/level13#210] [Q&A] [answered]
Does level 9 have a camp?

**Comment by nroutasuo** [comment, marked as answer]
Depends on your world seed. Look at the description in the log when you enter the level, it will usually contain a hint about whether the level is populated or not. Also check the level icon at the top of the page. It will look slightly different depending on if the level is habitable or not.

**Reply by Kiwithecoder** [reply]
Can you put two pictures side to side on whether it's populated or not? thanks!

**Reply by demoncoyote80** [reply]
this means it is uninhabited 
<img width="46" alt="Screenshot 2024-10-25 at 8 48 53 AM" src="https://github.com/user-attachments/assets/c1619bcd-6fc4-44b0-b15f-68f4a4ab24fd">
